libs/base/Data/Nat.idr

```diff
@@ -586,6 +627,18 @@ sucMinR : (l : Nat) -> minimum l (S l) = l
 sucMinR Z = Refl
 sucMinR (S l) = cong S $ sucMinR l
 
+-- Proofs on lte -----------------------
+
```

Comment:
Longer term (i.e. in a different PR), it is probably a good idea to think about introducing

```idris
data Reflects : Type -> Bool -> Type where
  True : p -> Reflects p True
  False : Not p -> Reflects p false
```

This way with very similar code we should be able to prove both that
`lte a b = True -> LTE a b` but also that `lte a b = False -> Not (LTE a b)`.
And have generic results on `Reflects` that makes proving these kind of properties
easier (e.g. the fact that `Not (LTE a b) -> LT b a` should be enough to get
`notlteIsLT` from `lteIsLTE`).

Reply:
> `False : Not p -> Reflects p false`

This might be slightly dangerous ;)

Reply:
Ugh. I tried to be thorough in my renaming of the Agda code I copy/pasted :(

Reply:
Done now, pending PR #722